test/dynamo/test_triton_kernels.py

```diff
@@ -1222,6 +1222,40 @@ def add_4_times_kernel(
             ["out_ptr"],
         )
 
+    @make_mutation_test
+    def test_labels():
+        @triton.jit
+        def kernel_with_label(
+            in_ptr0,
+            in_ptr1,
+            out_ptr,
+            n_elements,
+            BLOCK_SIZE: "tl.constexpr",
+        ):
+            pid = tl.program_id(axis=0)
+            if pid > 1:
+                return
```

Comment:
Should we add another test where we do different things in `if` and `else` blocks with a data-dependent condition like this?

Reply:
I already have a test for that which turns into a `scf.if` not labels

Reply:
Interesting, so labels are only used on `if ...: return`? or also `if` can have a body and this is still generated as label?

Reply:
Depends on what happens in the body, is it dynamic or not